dashboard/config/scripts_json/vpl-csd-2021.script_json

```diff
@@ -25,7 +26,7 @@
       "version_year": "2021"
     },
     "new_name": null,
-    "family_name": null,
+    "family_name": "vpl-csd",
```

Comment:
similar issue here, there is a vpl-csd-2020.script_json which does not have is_course set, so I would suggest making these consistent rather than more different